docs/install.md

```diff
@@ -0,0 +1,30 @@
+# MlAgility Installation Guide
+
+The following describes how to install MlAgility.
+
+## Installing mlagility locally
+
+Install the `mlagility` package into your environment simply run
+
+```
+pip install -e .
+```
+
+inside the mlagility repository.
+
+## Installing Slurm support
```

Comment:
Say what slurm is and why someone would want to install slurm support

Reply:
And link to the `--use-slurm` documentation as well

Reply:
Updated!